docs/paper/paper.md

```diff
@@ -31,19 +31,19 @@ Species occurrence data—georeferenced point locations where a species has been
 
 # Why Maxent still matters
 
 The main scientific contribution of `elapid` is extending and modifying the Maxent SDM, a model and software system as popular as it is maligned [@Phillips:2008; @Fourcade:2018]. First published in 2006, Maxent remains relevant because it's a presence-only model designed to work with the kinds of species occurrence data data that have proliferated lately.
 
-Presence-only models formulate binary classification models as presence/background (1/0) instead of presence/absence, which changes how models are fit and interpreted [@Fithian:2013; @Merow:2013]. Background points are a spatially-random sample of the landscapes where a species might be found, which should be sampled with the same level of effort and bias as the species occurrence data. Presence/background models posit the null expectation is that a species is equally likely to be found anywhere within it's range. Differences in environmental conditions between where a species occurs and in the conditions across the full landscape should indicate niche preferences. Relative habitat suitability is then determined based on differences in the relative frequency distributions of conditions in these regions. Presence-only models reduce the burden of finding absence data, which are problematic to boot, but they increase the burden of precisely selecting background points. These define what relative habitat suitability is defined as relative to [@Elith:2011; @Barbet:2012].
+Presence-only models formulate binary classification models as presence/background (1/0) instead of presence/absence, which changes how models are fit and interpreted [@Fithian:2013; @Merow:2013]. Background points are a spatially-random sample of the landscapes where a species might be found, which should be sampled with the same level of effort and bias as the species occurrence data. Presence/background models posit the null expectation that a species is equally likely to be found anywhere within it's range. Differences in environmental conditions between where a species occurs and across the full landscape should indicate niche preferences. Relative habitat suitability is then determined based on differences in the relative frequency distributions of conditions in these regions. Presence-only models reduce the burden of finding absence data, which are problematic to boot, but they increase the burden of precisely selecting background points. These define what relative habitat suitability is relative to [@Elith:2011; @Barbet:2012].
```

Comment:
Maybe saying "problematic to boot" sounds a bit too informal here?

Reply:
yes, i agree. I'll accept the changes in this PR as-is then update this wording on the main manuscript when i take another pass at it.